importAPI/put_job_test.go

```diff
@@ -22,20 +23,26 @@ func TestUpdateImportJobState(t *testing.T) {
 		Update:     validCreatedImportJobData,
 	}
 
-	if err := mongo.Setup(importJob); err != nil {
+	instance := &mongo.Doc{
+		Database:   cfg.MongoDB,
+		Collection: "instances",
+		Key:        "id",
+		Value:      instanceID,
+		Update:     validCreatedInstanceData,
+	}
+
+	if err := mongo.Setup(importJob, instance); err != nil {
 		log.ErrorC("Failed to set up test data", err, nil)
 		os.Exit(1)
 	}
 
 	importAPI := httpexpect.New(t, cfg.ImportAPIURL)
 
-	// These tests needs to refine when authentication was handled in the code.
-	// TODO Dont skip test once endpoint has been refactored
-	SkipConvey("Given an import job exists", t, func() {
+	Convey("Given an import job exists", t, func() {
 		Convey("When a request to update the jobs state with a specific id and the user is authenticated", func() {
 			Convey("Then the response returns status OK (200)", func() {
 
-				importAPI.PUT("/jobs/{id}", jobID).WithHeader(internalToken, internalTokenID).
+				importAPI.PUT("/jobs/{id}", jobID).WithHeader(tokenName, tokenSecret).
```

Comment:
This test seems to be failing with 
```
Error:      	
	            	expected status equal to:
	            	 "200 OK"
	            	
	            	but got:
	            	 "500 Internal Server Error"
```

Reply:
I tried rerunning the test on my macbook, I don't seem to be getting this error. Are you running the recipe-api and dataset-api(with `export MONGODB_DATABASE=test` )?